src/selectors/getCourses.js

```diff
@@ -0,0 +1,11 @@
+export default function getCourses(state) {
+  const courses = state.getIn([
+    'ui',
+    'assignmentCreationInterface',
+    'courses',
+  ]);
+  if (courses) {
+    return courses.toJS();
```

Comment:
`.toJS()` is considered harmful (main problem is it is slow) and I’m trying to remove everywhere we currently use it in the code base. Instead we should just make sure we are using `Record`s everywhere we should be (which as far as I can see we are in this PR) and then work with those directly in components etc. In many cases the interface is the same (you can do normal property access, `for…of` loops work with Immutable collections, etc.)